modules/programs/chromium.nix

```diff
@@ -156,6 +158,7 @@ in {
     google-chrome-dev =
       browserModule pkgs.google-chrome-dev "Google Chrome Dev" false;
     brave = browserModule pkgs.brave "Brave Browser" false;
+    vivaldi = browserModule pkgs.vivaldi "Vivaldi Browser" false;
```

Comment:
@rycee what is the reasoning for hiding other browsers from the documentation? (the last argument to `browserModule` controls the visibility)

Reply:
I believe it was just to avoid bloating the documentation with the same options over and over just with a slightly different attribute path. Maybe it's not too bad to allow generation of this documentation. Will have to try and see.